cmd/podman/varlink/io.podman.varlink

```diff
@@ -650,6 +655,10 @@ method PullImage(name: string) -> (id: string)
 #   "pod": "b05dee7bd4ccfee688099fe1588a7a898d6ddd6897de9251d4671c9b0feacb2a"
 # }
 #
+# $ varlink call unix:/run/podman/io.podman/io.podman.CreatePod '{"create": {"pause": true, "share": ["ipc", "net", "uts"]}}'
```

Comment:
"pause" should be "infra" to match "podman pod create --infra..."

Reply:
whoopsies, fixed